template/packages/types/src/Utils.ts

```diff
@@ -0,0 +1,5 @@
+export type DeepPartial<T> = T extends object
```

Comment:
This doesn't appear to be used anywhere. Is there any reason to keep it in the template?